Templates/DefaultComponent/Template/Source/${Name}Component.cpp

```diff
@@ -0,0 +1,72 @@
+// {BEGIN_LICENSE}
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+// {END_LICENSE}
+
+#include <${SanitizedCppName}Component.h>
+
+#include <AzCore/Serialization/SerializeContext.h>
```

Comment:
```suggestion
```
I don't think you need to include SerializeContext.h when EditContext.h already includes it for you